### ISSUE TITLE
Enhanced representation of gapped construction in English

### ISSUE THREAD

**Issue by sebschu** [UniversalDependencies/docs#411]
I'm currently struggling with providing the right analysis for the following sentence:

_Birds eat and sleep with their flock, not predators._

I would say that the basic representation should have the following dependencies:

`conj(eat, predators)`
`orphan(predators, not)`

But what about the enhanced representation? Should we have empty nodes for both verbs and their coordinating conjunction, and the case marker, as in the following example? Or should we do something simpler?

```
9    not ...      8 orphan    9.1:advmod|9.3:advmod
9.1  eat ...      _ _         2:conj
9.2  and ...     _ _          9.3:cc
9.3  sleep ...  _ _           9.1:conj:and
9.4  with ...    _ _          10:case
10   predators ... 2 conj     9.1:nmod:with|9.3:nmod:with
```

**Comment by jnivre** [comment]
First of all, I would vote for advmod(predators, not) instead of orphan(predators, not). The guidelines say that we don't use "orphan" for functional relations (like "cc", for example). I agree that "advmod" is borderline in general, but I would argue that negation should go with the functional relations.

As for empty categories, I would advocate a minimal approach, which in this case just involves inserting a predicate node, that is:

9    not ...      10 advmod    9.1:advmod
9.1  _ ...      _ _         2:conj
10   predators ... 2 conj     9.1:nmod:with

This empty node will (in some sense) be co-referential with the coordinated predicate in the preceding clause ("eat and sleep"). As for the case marker, I would suggest that we don't insert empty nodes for any element that would not trigger an orphan relation itself (in particular, leaf nodes).

Finally, is it really clear that the intended meaning is that birds don't eat and sleep with predators. Couldn't it instead mean that predators don't eat and sleep with their flock?

**Comment by dan-zeman** [comment]
+1 on advmod(predators, not).

It was my first reading that predators don't eat and sleep with their flock.

Finally, while I do see the missing predicate there, in this case I would feel sort of OK with just conj(birds, predators), no empty nodes involved. Consider 1. (both) birds and predators sleep; 2. birds but not predators sleep; 3. bird unlike predators sleep; 4. birds, not predators, sleep... etc.

**Comment by dseddah** [comment]
Hi all,
> Le 1 févr. 2017 à 10:01, Joakim Nivre <notifications@github.com> a écrit :
>  
> Finally, is it really clear that the intended meaning is that birds don't eat and sleep with predators. Couldn't it instead mean that predators don't eat and sleep with their flock?

at first I  got the second reading (predators don’t eat…).  Mb because of my non-nativeness I have troubles to get the first reading because of the  « with » ellipsis as in 
 
1a)   I  like eating with Mary, not John   (-> john doesn’t like eating w/ M)
  b)   I like eating with Mary, not with John (—> I don’t like eating w/ J)

are both reading possible with 1a too?

Djamé

**Comment by sebschu** [comment]
You all confused me :)  But based on the context and on native speaker judgements, the intended reading is really that birds don't sleep and eat with predators. (Though one of our syntacticians who is a native Bulgarian had the same intuition as the three of you.)

And I'll go with Dan's suggestion to not have any empty nodes. I think we should only insert empty nodes if there are `orphan` dependencies in the basic representation, so if we treat the negation as an `advmod`, then I think we should not have any empty nodes.

Thanks for the quick feedback!

**Comment by sebschu** [comment]
I'll also collect some of these examples while I'm annotating the treebank and I'll add them to the documentation when I get a chance.

**Comment by dseddah** [comment]
while I'm here, is there a specific thread regarding enhanced dependencies? my colleagues and I are considering doing a enhanced dependency conversion of the French Treebank (already available within a graph-based deep syntax native scheme and soon available with UD 2.0 too)

**Comment by sebschu** [comment]
I don't think so but you are welcome to start a thread. And that sounds great, we would love to see that happening!
